File: app/src/main/java/com/gmail/kaminskysem/PersnalHelper/Timer/TimerActivity.java
package com.gmail.kaminskysem.PersnalHelper.Timer;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.gmail.kaminskysem.PersnalHelper.Main.FragmentMainScrollingImages;
import com.gmail.kaminskysem.PersnalHelper.R;

public class TimerActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_timer);
        //add timer fragment
        TimerTImerFragment timerFragment = new TimerTImerFragment();
        getSupportFragmentManager().beginTransaction()
                .replace(R.id.fragment_timer_container, timerFragment)
                .commit();

        // add fragment from main activity
        FragmentMainScrollingImages fragmentMainScrollingImages = new FragmentMainScrollingImages();
        getSupportFragmentManager().beginTransaction()
                .replace(R.id.fragment_main_image_container,fragmentMainScrollingImages)
                .commit();


    }
}

File: app/src/main/java/com/gmail/kaminskysem/PersnalHelper/Timer/TimerActivity.kt
package com.gmail.kaminskysem.PersnalHelper.Timer

import android.os.Bundle
import android.view.View
import androidx.activity.compose.setContent
import androidx.appcompat.app.AppCompatActivity
import androidx.compose.foundation.Image
import androidx.compose.foundation.layout.Box
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.foundation.layout.padding
import androidx.compose.ui.Modifier
import androidx.compose.ui.layout.ContentScale
import androidx.compose.ui.res.painterResource
import androidx.compose.ui.unit.dp
import androidx.compose.ui.viewinterop.AndroidView
import androidx.fragment.app.FragmentContainerView
import com.gmail.kaminskysem.PersnalHelper.R

class TimerActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            Box(modifier = Modifier.fillMaxSize()) {
                Image(
                    painter = painterResource(id = R.drawable.img_more_2),
                    contentDescription = null,
                    contentScale = ContentScale.Crop,
                    modifier = Modifier.fillMaxSize()
                )

                TimerTimerScreen()
            }
        }
    }

    override fun onResume() {
        super.onResume()
    }
}
